protos/storage.proto

```diff
@@ -76,15 +76,15 @@ message WriteEntry {
   repeated FieldGroup field_groups = 2;
 }
 
-message WriteMetric {
-  string metric = 1;
+message WriteTable {
+  string table = 1;
   repeated string tag_names = 2;
   repeated string field_names = 3;
   repeated WriteEntry entries = 4;
 }
 
 message WriteRequest {
-  repeated WriteMetric metrics = 1;
+  repeated WriteTable tables = 1;
```

Comment:
Maybe we should call it `table_requests` if we modify the `WriteTable` to `WriteTableRequet`.

Reply:
Shouldn't the protocol be included in the name? Like：
`pointWriteRequest` or `SqlRequest`